docs/build/build-options.md

```diff
@@ -17,6 +17,36 @@ Additional information on building for Windows is available in the
 document and in the [SRT CookBook](https://srtlab.github.io/srt-cookbook/getting-started/build-on-windows/).
 
 
+## Building as a subproject
+
+CMake tool offers adding a complete project as a subdirectory. Variables used by the SRT
+project in this case remain in their own scope, but all variables from the parent scope
+are reflected. In order to prevent name clashes for option-designating variables, SRT provides
+a namespace-like prefixing for the optional variables it uses. If you want to configure
+optional variables from the level of `CMakeLists.txt` of the parent project, use the
+`LIBSRT_` prefix for the option names.
+
+This will not prevent the variables from being seen as derived in SRT project
+scope, but if you set explicitly a variable this way, it will be set to the
+desired value inside the SRT project, it will not set the same variable in the
```

Comment:
```suggestion
scope, but if you explicitly set a variable this way, it will be set to the
desired value inside the SRT project. It will not set the same variable in the
```